felles/src/main/kotlin/no/nav/familie/kontrakter/felles/dokarkiv/v2/ArkiverDokumentRequest.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.familie.kontrakter.felles.dokarkiv.v2
+
+import no.nav.familie.kontrakter.felles.dokarkiv.AvsenderMottaker
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotEmpty
+
+data class ArkiverDokumentRequest(@field:NotBlank val fnr: String,
```

Comment:
Dette ser ut å være bakåtkompatible endringer, trenger man en V2?

Reply:
Eller ja, `@field:NotEmpty val dokumenttype: Dokumenttype)` er vel breakig.

Reply:
Hvis det er éneste breaking - har vi noe som journalfører UTEN hoveddokumenter ? Det virker ulogisk for meg iallefall, men mulig jeg ikke har full oversikt. 

Hvis vi alltid journalfører MED hoveddokumenter så burde det vel ikke breaking i praksis?